include/host_alerts/DomainNamesConnectionAlert.h

```diff
@@ -0,0 +1,45 @@
+/*
+ *
+ * (C) 2013-21 - ntop.org
+ *
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ *
+ */
+
+#ifndef _DOMAIN_NAMES_CONNECTION_ALERT_H_
+#define _DOMAIN_NAMES_CONNECTION_ALERT_H_
+
+#include "ntop_includes.h"
+
+class DomainNamesConnectionAlert : public HostAlert {
+ private:
+  u_int32_t num_domain_names;
+  u_int8_t domain_names_threshold;
```

Comment:
255 as maximum value is too low. Think to a web server but also as a normal host doing normal activities.